packages/grafana-ui/src/components/Tooltip/Popover.tsx

```diff
@@ -1,96 +1,102 @@
-﻿import { Placement, VirtualElement } from '@popperjs/core';
-import React, { PureComponent } from 'react';
-import { Manager, Popper as ReactPopper, PopperArrowProps } from 'react-popper';
-import Transition from 'react-transition-group/Transition';
+﻿import {
+  FloatingArrow,
+  arrow,
+  autoUpdate,
+  flip,
+  offset,
+  shift,
+  useFloating,
+  useTransitionStyles,
+} from '@floating-ui/react';
+import React, { useLayoutEffect, useRef } from 'react';
 
+import { useTheme2 } from '../../themes';
+import { getPlacement } from '../../utils/tooltipUtils';
 import { Portal } from '../Portal/Portal';
 
-import { PopoverContent } from './types';
-
-const defaultTransitionStyles = {
-  transitionProperty: 'opacity',
-  transitionDuration: '200ms',
-  transitionTimingFunction: 'linear',
-  opacity: 0,
-};
-
-const transitionStyles: { [key: string]: object } = {
-  exited: { opacity: 0 },
-  entering: { opacity: 0 },
-  entered: { opacity: 1, transitionDelay: '0s' },
-  exiting: { opacity: 0, transitionDelay: '500ms' },
-};
-
-export type RenderPopperArrowFn = (props: { arrowProps: PopperArrowProps; placement: string }) => JSX.Element;
+import { PopoverContent, TooltipPlacement } from './types';
 
 interface Props extends Omit<React.HTMLAttributes<HTMLDivElement>, 'content'> {
   show: boolean;
-  placement?: Placement;
+  placement?: TooltipPlacement;
   content: PopoverContent;
-  referenceElement: HTMLElement | VirtualElement;
+  referenceElement: HTMLElement;
   wrapperClassName?: string;
-  renderArrow?: RenderPopperArrowFn;
+  renderArrow?: boolean;
```

Comment:
i guess this is a breaking change? 🤔

Reply:
According to the next comment, I think so 😅

Reply:
I actually don't think this is a (runtime) breaking change. JSX will be coerced to boolean and render the default floating-ui arrow if specified.